pyterrier/pipelines.py

```diff
@@ -95,7 +97,17 @@ def _ir_measures_to_dict(
             metric = m.measure
             metric = rev_mapping.get(metric, str(metric))
             rtr[m.query_id][metric] = m.value
+        if backfill_qids is not None:
```

Comment:
It was necessary to move the backfilling stuff down to this level where rev_mapping is available. It was actually buggy before: If a family name was given and perquery was set, the family name would appear with a 0 in the summary, rather than the member names. Tests catch this now.